src/terminal/terminal/terminal-priv.h

```diff
@@ -656,5 +666,27 @@ void guac_terminal_copy_rows(guac_terminal* terminal,
  */
 void guac_terminal_flush(guac_terminal* terminal);
 
+/**
+ * Determination of part of word
+ * Match like this pattern : [0-9\/A-Za-z_\~\&\-\.\?\\]
+ */
+bool guac_terminal_is_part_of_word(int ascii_char);
+
+/**
+ * Determination of blank
+ * Match like this pattern : [\0\ ]
+ */
+bool guac_terminal_is_blank(int ascii_char);
+
+/**
+ * Get the char (int ASCII code) at the row/col requested.
+ */
+int guac_terminal_get_char(guac_terminal* terminal, int row, int col);
+
+/**
+ * Selection on double click event
+ */
+void guac_terminal_double_click(guac_terminal* terminal, int row, int col);
+
```

Comment:
I don't think any of these functions are used outside of the `terminal.c` file, correct? If so, there's no reason to put them in the header file - you can just define them in the source file and also mark them as `static`.

Also, for the documentation for the functions - whether here or in the `termianl.c` file - you'll need to do more complete documentation - should include the overall description, the parameters, and the return value (if any).

Reply:
- Declarations removed from header file.
- Functions marked as `static`.
- Function documentation moved to `terminal.c`.
- Improved documentation.